app/controllers/people_controller.rb

```diff
@@ -226,7 +226,7 @@ def destroy
       MarketplaceService::Listing::Command.delete_listings(@current_user.id)
 
       communities.each { |community_id|
-        PaypalService::API::Api.accounts_api.delete(community_id: @current_community.id, person_id: @current_user.id)
```

Comment:
Interesting. This is wrong on multiple levels. We shouldn't be deleting accounts connected to other communities and also there's the bug of not using the iterated over collection contents for anything.